frontend/benefit/applicant/src/components/mainIngress/MainIngress.tsx

```diff
@@ -16,12 +17,25 @@ import {
 import { useMainIngress } from './useMainIngress';
 
 const MainIngress: React.FC = () => {
-  const {
-    handleNewApplicationClick,
-    handleMoreInfoClick,
-    t,
-  } = useMainIngress();
+  const { handleNewApplicationClick, handleMoreInfoClick, t } =
+    useMainIngress();
   const { errors } = React.useContext(FrontPageContext);
+  const { submittedApplication, setSubmittedApplication } =
+    React.useContext(AppContext);
+
+  const successNotification = submittedApplication ? (
+    <$Notification
+      label={t('common:notifications.applicationSubmitted.label')}
+      type="success"
+      dismissible
+      onClose={() => setSubmittedApplication(null)}
+    >
+      {t('common:notifications.applicationSubmitted.message', {
+        applicationNumber: submittedApplication?.applicationNumber,
+        applicantName: submittedApplication?.applicantName,
+      })}
+    </$Notification>
+  ) : null;
```

Comment:
Thinking about the logic here, this success notification is based on the frontend assumption that the request was successful without waiting for the actual backend response. Or am I missing something?
If so, what if the request for some reason failed.